.github/workflows/deploy-client.yml

```diff
@@ -28,7 +28,12 @@ jobs:
 
   deploy:
     runs-on: ubuntu-latest
-    if: github.ref == 'refs/heads/main'
+    if: >
```

Comment:
Should we also check for the case when the PR is merged?
```
if: >
    github.event.pull_request.merged == true
```

Reply:
I don't think we need to because we have the branch check.